src/Backend/Snowflake/SnowflakeImportAdapterInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Db\ImportExport\Backend\Snowflake;
+
+use Keboola\Db\Import\Snowflake\Connection;
+use Keboola\Db\ImportExport\Backend\BackendImportAdapterInterface;
+
+interface SnowflakeImportAdapterInterface extends BackendImportAdapterInterface
+{
+    public function __construct(Connection $connection);
```

Comment:
Je to k něčemu dobrý definovat konstruktor v interface? Přijde mi to jako implementační detail té classy která ho implementuje co potřebuje za závislosti.

Reply:
Už to vidím, je to kvůli tomu dynamickému vytváření v `setUpAdapter`, ok.